src/api/goods.ts

```diff
@@ -6,6 +6,16 @@ export function getAll(): Promise<Good[]> {
     .then(response => response.json());
 }
 
-export const get5First = () => {};
+export const get5First = ():Promise<Good[]> => {
+  return fetch(API_URL)
+    .then(response => response.json())
+    .then(goodsFromServer => goodsFromServer
+      .filter((good: { id: number; }) => good.id <= 5));
```

Comment:
Unless I'm wrong, in function 'get5first' goods, should be sorted by name (first point of the checklist)